ironmon_tracker/data/PokemonData.lua

```diff
@@ -158,6 +160,11 @@ function PokemonData.initialize()
 					pokemonData.abilities = abilities
 				end
 			end
+			-- read in base friendship from Rom file for all mons - Tainted_wolf
+			local friendshipBase = PokemonData.readPokemonBaseFriendshipFromMemory(pokemonID)
+			if friendshipBase ~= nil then
```

Comment:
Doesn't this contradict with the types and abilities, which are memory read functions that then have a nil check immediately after in this same codeblock?

Reply:
You are correct. The nil checks for both of those are more-or-less useless, since the return value of their respective functions always returns a table with 2 entries. It can't ever be nil. I'll update `dev` with this fix after the PR merger.